### ISSUE TITLE
Destruction: Blast chunks .fbx assets fail to be processed by the Asset Processor due to the “python: No EBus connection detected; missing call or failed call to connect()?” error

### ISSUE THREAD

**Issue by LB-MateuszSzweda** [o3de/o3de#13362]
**Describe the bug**
All of the available _pab_box_2_chunks.fbx_, _box_2_chunks.fbx_, _cinder_wall_complex.fbx_ blast chunks assets are processing through the Asset Processor with the following error:

`python: No EBus connection detected; missing call or failed call to connect()?`

It was also observed that the same error shows in the console when any of the above assets is tried to be open with **FBX Settings** tool in the Editor.

As a result of the above blast chunks assets are not present in the Asset picker of **Blast Family Mesh Data** component and can not be picked.

The issue occurs in default AutomatedTesting project and any project that has the above assets migrated.

For further information please refer to the attached screenshot, Asset Processor logs and blast chunks assets.

**Assets required**
_pab_box_2_chunks.fbx_ , _box_2_chunks.fbx_ or _cinder_wall_complex.fbx_ blast chunks assets (they can be found in AutomatedTesting _\Assets\Destruction_ directory)

**Prerequisites**

* **NVIDIA Blast**, **Python Asset Builder**, **Editor Python Bindings** Gems are enabled and built for the active project.
* _pab_box_2_chunks.fbx_ , _box_2_chunks.fbx_ or _cinder_wall_complex.fbx_ blast chunks assets were moved into the active project directory.

**Steps to reproduce**

1. Launch the Editor and create a new level.
2. Create a new entity.
3. Add **Blast Family** and **Blast Family Mesh Data** components to it.
4. Provide the **Blast Family** component with the _cinder_wall_complex.blast_ asset.
5. Try to pick any asset for **Blast Family Mesh Data** component with its asset picker window.
6. Open the Asset Browser and try to open _cinder_wall_complex.fbx_ asset in **Fbx Settings** tool.

**Expected behavior**
Blast chunks .fbx assets are detected and are able to be picked for **Blast Family Mesh Data** component and opened in **Fbx Settings** tool.

**Actual behavior**
Blast chunks .fbx assets are unable to be detected and picked for **Blast Family Mesh Data** component and opening them in **Fbx Settings** tool results in `python: No EBus connection detected; missing call or failed call to connect()?` error.

**Screenshots/Video**
![destruction chunk assets errors in AP](https://user-images.githubusercontent.com/86953456/203554496-4cdd20da-5b00-483f-bc50-c05b8fe98be1.png)

**Found in Branch**
Development (7cb2c3f)

**Desktop/Device:**
- Device: PC
- OS: Windows
- Version: 10 Pro
- CPU: AMD Ryzen 9 3900X
- GPU: NVIDIA GeForce RTX 2060 SUPER
- Memory: 16GB

**Additional context**
[cinder_wall_complex.fbx-380023773-13738.log](https://github.com/o3de/o3de/files/10075795/cinder_wall_complex.fbx-380023773-13738.log)
[pab_box_2_chunks.fbx-150828949-13598.log](https://github.com/o3de/o3de/files/10075801/pab_box_2_chunks.fbx-150828949-13598.log)
[box_2_chunks.fbx-982310958-13752.log](https://github.com/o3de/o3de/files/10075802/box_2_chunks.fbx-982310958-13752.log)
[blast chunks fbx assets.zip](https://github.com/o3de/o3de/files/10075811/blast.chunks.fbx.assets.zip)

**Comment by hultonha** [comment]
Tagging @o3de/sig-content 

We are not aware of any changes that have happened to Blast from a Physics/sig-simulation perspective so we think this failure may have been caused by a recent change on the asset side. We are also surprised this failure has not caused a failure in AR which should have caught this. Tagging @lmbr-pip for visibility who might be able to ping the relevant people who'd know what's caused this. Thanks!

**Comment by hultonha** [comment]
@LB-MateuszSzweda - could you please try clearing your cache and attempting to repro to see if this happens again or was it a build artifact issue of some sort? Thanks!

**Comment by LB-MateuszSzweda** [comment]
@hultonha  I was able to reproduce it today on Development (92af95a) and the issue persisted after clearing the cache and reprocessing the assets.

**Comment by hultonha** [comment]
Understood, thanks for checking @LB-MateuszSzweda! 👍

**Comment by AMZN-stankowi** [comment]
I can look at this after I finish up a couple other tickets. In front of this I have: periodic build configuration failure with AP tests, and Editor fails to launch if AP / Editor is shut down before processing all critical assets.

**Comment by hultonha** [comment]
> I can look at this after I finish up a couple other tickets. In front of this I have: periodic build configuration failure with AP tests, and Editor fails to launch if AP / Editor is shut down before processing all critical assets.

That would be fantastic thank you @AMZN-stankowi, please let us know if there's anything we can do to help. Much appreciated!

**Comment by AMZN-stankowi** [comment]
https://github.com/o3de/o3de/pull/13607

**Comment by hultonha** [comment]
> #13607

Amazing thanks @AMZN-stankowi! 🤩

**Comment by AMZN-stankowi** [comment]
The only blast FBX I could find was test_chunks.fbx, and it used the test chunks builder python script. I fixed that here https://github.com/o3de/o3de/pull/13607
I upgraded the error message to describe the ebus that was called out of order, so I would send any remaining issues to the team that maintains the python script associated with any FBX files with issue. The error should be descriptive enough to address now.
